addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -524,7 +527,7 @@ export class HtmlField extends Component {
         $odooEditor.removeClass('odoo-editor-editable');
         $editable.html(html);
 
-        await toInline($editable, this.cssRules, this.wysiwyg.$iframe);
+        simpleToInline($editable, this.cssRules, this.wysiwyg.$iframe);
```

Comment:
Ah, now I remember the `async` thing about toInline. But note that it is not really an issue, because an async function will actually be called synchronously until it hits an `await`. So if the lines with `await` are all in conditional blocks that are not used in this context (through options for example), then the entire body of the function will execute synchronously, even if it is marked as `async`.

Reply:
Indeed, we will see the result of https://github.com/odoo/odoo/pull/115391#discussion_r1177766914 discussion, but for now the `simpleToInline` doesn't have async component anyways.